oe_theme.theme

```diff
@@ -1023,6 +1023,20 @@ function oe_theme_preprocess_block(&$variables) {
   ]);
 }
 
+/**
+ * Implements template_preprocess_block__system_branding_block().
+ *
+ * For preprocessing default mode of System Branding block.
```

Comment:
Not sure this comment describes what we are doing here, I think something like "Add required variables to follow ECL guidelines" or something similar would work better.

Reply:
Thanks.